src/security/Session.h

```diff
@@ -18,6 +18,31 @@
 #if USE_OPENSSL
 #if HAVE_OPENSSL_SSL_H
 #include <openssl/ssl.h>
+
+/// These are not included by openssl/ssl.h when deprecated API is disabled
```

Comment:
These includes should be added to the .h or .cc where the symbols they define are needed. Prefer the src/security/forward.h if there is a spread of usage.